src/bundle/Resources/public/js/scripts/helpers/browser.helper.js

```diff
@@ -0,0 +1,7 @@
+const { userAgent } = window.navigator;
+const isChrome = () => userAgent.indexOf('Chrome') != -1;
```

Comment:
String has `includes()` method, which is superior `indexOf` :)
```suggestion
const isChrome = () => userAgent.includes('Chrome');
```